# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/serviceworkers/PFetchEventOpProxy.ipdl
# Commit: 8fec60d0c499
# Full Hash: 8fec60d0c499fdf429f4d11da6bea0c65aecb1d4
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2022-02-27 09:40:54
# Regressor Bug: 1754365
# File Overlap Count: 1
# Description:
#   Bug 1754365 - Create IPC for transferring NavigationPreload's PerformanceTimingData. r=dom-worker-reviewers,jesup
#   
#   Depends on D138250
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138251
# ==============================================================================

diff -r fb0ac558803d -r 8fec60d0c499 dom/serviceworkers/PFetchEventOpProxy.ipdl
--- a/dom/serviceworkers/PFetchEventOpProxy.ipdl	Sun Feb 27 00:26:21 2022 +0000
+++ b/dom/serviceworkers/PFetchEventOpProxy.ipdl	Sun Feb 27 00:26:22 2022 +0000
@@ -23,7 +23,7 @@
   async __delete__(ServiceWorkerFetchEventOpResult aResult);
 
  child:
-  async PreloadResponse(ParentToChildInternalResponse aResponse);
+  async PreloadResponse(ParentToChildResponseWithTiming aResponse);
 };
 
 }  // namespace dom